spec/plans/discover.fmf

```diff
@@ -105,108 +91,9 @@ description: |
-            tests containing at least one of them are selected.
-            Regular expressions are supported for both relation
-            and target. Relation part can be omitted to match all
-            relations.
-        filter
-            Apply advanced filter based on test metadata
-            attributes. See ``pydoc fmf.filter`` for more info.
-        exclude
-            Exclude tests which match a regular expression.
-        prune
-            Copy only immediate directories of executed tests and
-            their required files.
-
-            .. versionadded:: 1.29
-
-        It is also possible to limit tests only to those that have
-        changed in git since a given revision. This can be
-        particularly useful when testing changes to tests
-        themselves (e.g. in a pull request CI). Related config
-        options (all optional) are:
-
-        modified-only
-            Set to ``true`` if you want to filter modified tests
-            only.  The test is modified if its name starts with
-            the name of any directory modified since modified-ref.
-        modified-url
-            An additional remote repository to be used as the
-            reference for comparison. Will be fetched as a
-            ``reference`` remote in the test dir.
-        modified-ref
-            The ref to compare against. Defaults to the local
-            repository's default branch. Note that you need to
-            specify ``reference/<branch>`` to compare to a branch
-            from the repository specified in ``modified-url``.
+        available tests in given repository.
 
-        Use the :ref:`/spec/plans/discover/dist-git-source`
-        options to download rpm sources for dist-git repositories.
-        In addition to the common dist-git-source features, the
-        ``fmf`` plugin also supports the following options:
-
-        dist-git-init
-            Set to ``true`` to initialize fmf root inside
-            extracted sources at ``dist-git-extract`` location or
-            top directory. To be used when the sources contain fmf
-            files (for example tests) but do not have an
-            associated fmf root.
-
-        dist-git-remove-fmf-root
-            Set to ``true`` to remove any fmf root present in the
-            sources.  ``dist-git-init`` can be used to create it
-            later at desired location.
-
-        dist-git-merge
-            Set to ``true`` to combine fmf root from the sources
-            and fmf root from the plan.  It allows to have plans
-            and tests defined in the DistGit repo which use tests
-            and other resources from the downloaded sources. Any
-            plans in extracted sources will not be processed.
-
-        dist-git-extract
-            Path specifying what should be copied from the
-            sources.  Defaults to top fmf root or top directory
-            ``/``.
+        See the `fmf identifier`_ documentation for details.
 
```

Comment:
So now here remains just short info about the plugin and a couple of examples. Nice! What about including here a reference to the fmf plugin documentation so that users know where to look for details?

Reply:
@psss Reference added !